components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/AbstractIdentityUserMgtFailureEventListener.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
AFAIK, this space shouldn't be there in the license. [This](https://www.apache.org/licenses/LICENSE-2.0#apply) is how Apache official license looks like. Although I couldn't find any official WSO2 resources on this except other code files. IMHO license should be as follows.
```
/*
 * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
 *
 * WSO2 Inc. licenses this file to you under the Apache License,
 * Version 2.0 (the "License"); you may not use this file except
 * in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing,
 * software distributed under the License is distributed on an
 * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 * KIND, either express or implied.  See the License for the
 * specific language governing permissions and limitations
 * under the License.
 */
```